api/user.proto

```diff
@@ -170,11 +170,13 @@ message UserSessionIdentifiers {
 }
 
 message UserSession {
-  UserIdentifiers user_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
-  string session_id = 2 [(gogoproto.customname) = "SessionID", (validate.rules).string.max_len = 64];
+  UserIdentifiers user_ids = 1 [(gogoproto.customname) = "UserIDs", (gogoproto.nullable) = false, (validate.rules).message.required = true];
```

Comment:
Nit: what's the reason for the custom name here/the removal of embed ? It makes the change set longer than it could have been.

Reply:
I wanted to bring the structure closer to the one of access tokens. But yeah, I guess in the end it's an unnecessary change, so I reset it to the original structure.